components/chat_ui/resources/page/chat_ui.html

```diff
@@ -0,0 +1,30 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>Chat Summarizer UI</title>
+    <link rel="stylesheet" href="chrome-untrusted://resources/brave/css/reset.css">
+    <link rel="stylesheet" href="chrome-untrusted://resources/css/text_defaults.css">
+    <link rel="stylesheet" href="chrome-untrusted://resources/brave/fonts/poppins.css">
+    <script src="chrome-untrusted://resources/js/load_time_data_deprecated.js"></script>
+    <script src="/strings.js"></script>
+    <style>
+      body {
```

Comment:
can this be moved to a .css file to avoid having inline-styles in the CSP?

Reply:
I have moved this to a separate css file